### ISSUE TITLE
Person detection training doesn't converge. Evaluation showing accuracy of 52% after 1M steps

### ISSUE THREAD

**Issue by sambhusuryamohan** [tensorflow/tensorflow#42086]
@tensorflow/micro

**System information**
- Host OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- TensorFlow installed from (source or binary): Docker installation
- Tensorflow version (commit SHA if source): 1.14.0
- Target platform (e.g. Arm Mbed OS, Arduino Nano 33 etc.): None

**Describe the problem**
I have done the training as exactly as in the document https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/examples/person_detection/training_a_model.md with some minor command changes which is not updated in the original document. The training was done for 1Million steps and when evaluated the accuracy was 52%. 
The minor modifications were 
1. use the script download_and_convert.py instead of build_visualwakewords_data.py. 
2. Changed arguments --input_grayscale to --use_grayscale

**Please provide the exact sequence of commands/steps when you ran into the problem**
The commands i used are as follows all from the path models/research/slim
1. python slim/download_and_convert_data.py \
    --dataset_name=visualwakewords \
    --dataset_dir=/tmp/visualwakewords

2. python slim/train_image_classifier.py     --train_dir=vww_96_grayscale     --dataset_name=visualwakewords     --dataset_split_name=train     --dataset_dir=coco_dataset/     --model_name=mobilenet_v1_025     --preprocessing_name=mobilenet_v1     --train_image_size=96     --use_grayscale=True     --save_summaries_secs=300     --learning_rate=0.045     --label_smoothing=0.1     --learning_rate_decay_factor=0.98     --num_epochs_per_decay=2.5     --moving_average_decay=0.9999     --batch_size=96     --max_number_of_steps=1000000

3. python slim/eval_image_classifier.py --alsologtostderr checkpoint_path=vww_96_grayscale/model.ckpt-1000000 --dataset_dir=coco_dataset/ --dataset_name=visualwakewords --dataset_split_name=val --model_name=mobilenet_v1_025 --preprocessing_name=mobilenet_v1 --use_grayscale=True --train_image_size=96

Can someone help me in identifying why it is not converging as stated in the paper?

**Comment by sambhusuryamohan** [comment]
Sorry my mistake there was a typo in checkpoint_path argument. Now I am getting 81%. 
One additional change to training_a_model.md. Changed the argument --train_image_size to --eval_image_size.
